testdata/project-v2/.gitignore

```diff
@@ -13,10 +13,6 @@ bin
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Kubernetes Generated files - skip generated files, except for vendored files
-
-!vendor/**/zz_generated.*
-
```

Comment:
Why it was removed from the .gitignore ?

Reply:
The `.gitignore` file was generated exactly the same for `v1` and `v2`. Now they are scaffolded separately. As `v1` uses GOPATH and vendor, that line is needed for `v1`. But `v2` used GOMODULES which doesn't have a `vendor` folder.

Reply:
This change make sense, however, IHMO the extraction of any scaffold feature which has been shared between the versions should be done in a specific PR for each in order to make easier the review, track the changes and revert if be required.